subprojects/packagefiles/pegtl/meson.build

```diff
@@ -0,0 +1,24 @@
+project(
+  'pegtl',
+  'cpp',
+  version: '3.2.7',
+  meson_version: '>=1.2.0',
+  default_options: {'cpp_std': 'c++17'},
+)
+
+pegtl_dep = declare_dependency(
+  include_directories: include_directories('include'),
+  version: meson.project_version(),
```

Comment:
`version` already defaults to the project version.

Reply:
Ok thanks, didn't know that 👍